src/Menu.js

```diff
@@ -59,7 +59,9 @@ module.exports = class extends React.Component {
       )
     )
       return;
-    this.context.ambManager.closeMenu();
+    if(!this.context.ambManager.options.closeOnSelection) {
```

Comment:
I'm not sure whether the code should be changed or the issue description.  Form the description, it seems as though you want the `closeMenu` to happen when `closeOnSelection` is true.  From the code, it looks as though it will only happen when `closeOnSelection` is false.

Reply:
Thanks for catching that. Silly mistake on my part. I believe if `closeOnSelection` is true then the `closeMenu` function should be called. Fixed